app/assets/javascripts/helpers.js

```diff
@@ -19,3 +19,34 @@ function linkToTemplatesOnGithub() {
     element.removeAttr('data-debug-template-path');
   });
 };
+
+function trackInternalLinks (rootSelector) {
+  rootSelector = rootSelector || 'body';
+  var currentHost = document.location.protocol + '//' + document.location.hostname;
+  var internalLinkSelector = 'a[href^="' + currentHost + '"], a[href^="/"]';
+
+  $(rootSelector).on('click', internalLinkSelector, trackClickEvent);
```

Comment:
This adds a listener to the root - I think what will be better is if you did:
```suggestion
  $(rootSelector).find(internalLinkSelector).on('click', trackClickEvent);
```